src/components/CouponDetails/index.jsx

```diff
@@ -626,7 +631,7 @@ class CouponDetails extends React.Component {
       assignments_remaining: `${code.redemptions.total - code.redemptions.used - code.redemptions.num_assignments}`,
       assignment_date: `${code.assignment_date}`,
       last_reminder_date: `${code.last_reminder_date}`,
-      is_public: `${code.is_public}`,
+      is_public: code.is_public ? 'Public' : 'Private',
```

Comment:
I'd rather have this be a boolean than strings, but it looks like you may have changed it from a boolean to not for what I'm sure is a good reason. But that tempts me to change the status to `publicStatus` rather than `isPublic`. 
But then it's not in line with the data from the backend, so, in conclusion, this is probably fine. I might leave a comment about why you're not leaving it as a Boolean.

Reply:
This is for display purposes, product request